Tests/Modules/MetadataModuleSpecs.swift

```diff
@@ -187,6 +208,67 @@ class MetadataModuleSpecs: QuickSpec {
                         }
                     }
                 }
+
+                it("should make API call to create metadata template with copyInstanceOnItemCopy and produce file model when API call succeeds") {
```

Comment:
Do we need another test or can we just merge this field into the test above?

Reply:
I'm thinking of keeping both tests to validate that the behavior works as expected both when copyInstanceOnItemCopy is passed in and when its not, since its just an optional field